package.json

```diff
@@ -33,7 +33,8 @@
     "testonly": "babel-node ./node_modules/.bin/_mocha $npm_package_options_mocha",
     "lint": "eslint src",
     "check": "flow check",
-    "build": "rm -rf lib/* && babel src --ignore __tests__ --out-dir lib",
+    "build": "rm -rf lib/* && babel src --ignore __tests__ --out-dir lib && cp package.json lib/ && npm run build:flow",
```

Comment:
I have no idea. I was cargo culting. I'll remove it unless you have an idea why it's necessary.